pkg/apis/pipeline/v1beta1/task_validation_test.go

```diff
@@ -285,7 +285,7 @@ func TestTaskSpecValidateError(t *testing.T) {
 		name: "nil",
```

Comment:
i know this isn't part of the diff but it's incorrectly named - the Task Spec does not end up being nil in this test.  It ends up being a valid pointer to an empty TaskSpec object.

It doesn't look like we exercise the nil case.

Reply:
I changed the name.